packages/commercetools/composables/src/useUser/authenticate.ts

```diff
@@ -3,10 +3,19 @@ import { CustomerSignMeInDraft, CustomerSignMeUpDraft } from '../types/GraphQL';
 
 type UserData = CustomerSignMeUpDraft | CustomerSignMeInDraft;
 
+const validateUserResponse = (response) => {
+  if (response?.data?.user) {
+    return response.data.user;
+  }
+
```

Comment:
remove empty line what do you think ? :)

Reply:
It's added on purpose to increase readability